File: Basic_Program/Factorial_recursive.java
import java.util.Scanner;

public class Factorial_recursive {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);

        System.out.println("Enter any integer number :");
        int n = sc.nextInt();

        Factorial_recursive obj = new Factorial_recursive();
        System.out.println("Factorial by recursive : " + obj.factorial(n));
    }

    int factorial(int n) {
        if (n == 1)
            return 1;
        else
            return n * (factorial(n - 1));
    }
}

File: Basic_Program/Factorial_recursive.c
#include <stdio.h>

int factorial(int n)
{
    if (n == 1)
        return 1;
    else
        return n * (factorial(n - 1));
}

void main()
{
    int n;

    printf("enter any integer number : ");
    scanf("%d", &n);

    printf("Factorial by recursive : %d",factorial(n));   
}